publify_core/app/assets/javascripts/spinnable.js

```diff
@@ -1,5 +1,5 @@
 // Show and hide spinners on Ajax requests.
 $(document).ready(function(){
   $('form.spinnable').on('ajax:before', function(evt, xhr, status){ $('#spinner').show();})
-  $('form.spinnable').on('ajax:after', function(evt, xhr, status){ $('#spinner').hide();})
+  $('form.spinnable').on('ajax:complete', function(evt, xhr, status){ $('#spinner').hide();})
```

Comment:
This is the correct solution, but this is the ONLY change that should be in this PR. 

Always create a new branch off `master` for each new PR, so it contains ONLY the fix for that PR's referenced issue.